x/wasm/ibc.go

```diff
@@ -267,23 +268,38 @@ func (i IBCHandler) OnRecvPacket(
 		return channeltypes.NewErrorAcknowledgement(errorsmod.Wrapf(err, "contract port id"))
```

Comment:
Is this something where we can provide a better error message than the redacted error too?

Reply:
Good 👁️ ! Unless we touch our contract-to-port mapping we should never run into this case. Either we `panic` here or we need to return the error event, too.
I would prefer panic to make clear this must not happen